js/spreadsheet.js

```diff
@@ -421,6 +421,24 @@ var spreadsheet = function(options) {
       }
     },
     beforePaste: function(data, coords) {
+      var cellsToSelect = [];
+
+      // Checks if the entire row is selected
+      if (getColWidths().length === coords[0].endCol - coords[0].startCol + 1) {
+        // Selects only start column of each row to prevent populating
```

Comment:
"Select first cell to prevent [...]"

Reply:
Changes the comment, but It selects first cell for each selected row to allow users copy/paste multiple rows.